subsys/bluetooth/Kconfig

```diff
@@ -178,4 +178,13 @@ config BT_COMPANY_ID
 
 source "subsys/bluetooth/mesh/Kconfig"
 
+config BT_DISABLE
+	bool "Enable Bluetooth disable functionality"
+	select DYNAMIC_DIRECT_INTERRUPTS if CONFIG_BT_LLL_VENDOR_NORDIC
+	help
+	  Enable the Bluetooth disable API which would stop all Bluetooth
+	  activity and reset the Bluetooth controller. If using the internal
+	  controller the interrupts will be released and the application can
```

Comment:
This Kconfig file is generic to Bluetooth, so we should not be mentioning the "internal" controller, IMO
Also, some parts of the text are not clear to me: which are the "interrupts" you refer to, could you add some clarification?